torch/jit/frontend.py

```diff
@@ -337,12 +337,12 @@ def build_Name(ctx, expr):
 
     @staticmethod
     def build_NameConstant(ctx, expr):
-        text = "True" if expr.value else "False"
-        r = ctx.make_range(expr.lineno, expr.col_offset, expr.col_offset + len(text))
-        if expr.value:
+        r = ctx.make_range(expr.lineno, expr.col_offset, expr.col_offset + len(str(expr.value)))
+        if expr.value is True:
             return TrueLiteral(r)
-        else:
+        elif expr.value is False:
             return FalseLiteral(r)
+        return Var(Ident(r, "None"))
```

Comment:
If you want to prevent errors like this one in the future, it would be better to also check that `expr.value is None` and otherwise make it a hard error.